src/Symfony/Component/Messenger/Bridge/Doctrine/composer.json

```diff
@@ -17,18 +17,14 @@
     ],
     "require": {
         "php": ">=8.0.2",
+        "doctrine/dbal": "^2.10|^3.0",
+        "doctrine/persistence": "^1.3|^2",
```

Comment:
the doctrine/persistence one is optional. Part of the bridge features are about integrating with DBAL only.
and doctrine/dbal is also optional. Some of the doctrine/persistence features can be used for the ODM for instance.

The root cause of the issue is that we should not have 1 doctrine bridge but 2.

Reply:
hm.. I do feel it would be too confusing to split it. So I'll revert my changes.

Reply:
This was the main reason I wanted to make this PR btw..

Reply:
Which features of the Doctrine bridge for Messenger can be used without DBAL? When would it make sense to use this bridge without it?

Reply:
ok, `symfony/doctrine-messenger` is actually only about the DBAL transport. So it indeed has a strong dependency on DBAL, and *no dependency* on persistence.

The messenger middlewares for DBAL and doctrine/persistence are in `symfony/doctrine-bridge`